app/src/main/java/eu/kanade/presentation/manga/components/MangaInfoHeader.kt

```diff
@@ -210,11 +212,15 @@ fun MangaActionRow(
     // SY <--
     modifier: Modifier = Modifier,
 ) {
+
+    val libraryPreferences: LibraryPreferences = Injekt.get()
+    val restrictions = libraryPreferences.autoUpdateMangaRestrictions().get()
+    val isSkipCompleted = MANGA_NON_COMPLETED !in restrictions
     val defaultActionButtonColor = MaterialTheme.colorScheme.onSurface.copy(alpha = DISABLED_ALPHA)
 
     // TODO: show something better when using custom interval
     val nextUpdateDays = remember(nextUpdate) {
-        return@remember if (nextUpdate != null) {
+        return@remember if (nextUpdate != null && isSkipCompleted) {
```

Comment:
if Skip is ticked, then this will make even other non-completed mangas will also not showing predicted time

Reply:
I've just noticed that mb
but if I add || status != SManga.COMPLETED.toLong() to isSkipCompleted, I would have to add a status: Long,
but when I add it, it makes MangaScreen have errors
how would I fix it?